console/app_serve.go

```diff
@@ -36,13 +37,14 @@ func (s AppServe) Handle(c inter.Cli) inter.ExitCode {
 		http.HandleHttpKernel(app, response, request)
 	}
 
-	c.Info("Start %s to handle requests", name)
+	c.Line("\u001B[32mStarting %s server:\u001B[0m http://%s", name, s.getAddr(c.App()))
```

Comment:
If address is not available, then this is not a valid url. Maybe at that moment you can show a completely different sentence, more like this: `Listens to all addresses with port 80, for example http://localhost:80`